conf/app.routes

```diff
@@ -46,3 +46,5 @@ GET       /we-cannot-find-your-application              controllers.CannotFindAp
 
 ### Switchover error page route
 GET       /error/problem-with-your-answers              controllers.SwitchOverErrorController.onPageLoad
+GET       /register-charity-hmrc/error/signed-you-out   controllers.SignOutController.signedYouOut
```

Comment:
please check the url, "/register-charity-hmrc" is not required here

Reply:
yep. the jira has been updated to remove the url path.

Reply:
done